wrangler/src/trust/trust.js

```diff
@@ -0,0 +1,271 @@
+import { Keypair, TransactionBuilder, Networks, BASE_FEE, Operation, Account } from 'stellar-base'
+import { parse } from 'toml'
+import localtoml from '../turret/toml'
+
+/**
+ * It gets the account info for a given public key.
+ * @param pubkey - The public key of the account you want to get information about.
+ * @returns The JSON response from the Horizon API.
+ */
+export async function getAccountInfo(pubkey){
+    const accountInfo = (await fetch(`${HORIZON_URL}/accounts/${pubkey}`))
+    return accountInfo.json()
+}
```

Comment:
we call this function for nearly every piece of  account info we need, likely resulting in excess Horizon calls.

Might make sense to write a `getAccount` function in `stellar-sdk-utils` that fetches and returns an accountResponse.

Said response could then be re-used throughout the majority of the utils

Reply:
yes it would be smart to cache the account info